package.json

```diff
@@ -35,7 +35,7 @@
     "@angular/platform-browser-dynamic": "^14.0.6",
     "@angular/platform-server": "^14.0.6",
     "@angular/router": "^14.0.6",
-    "@apollo/client": "3.7.5",
+    "@apollo/client": "3.7.7",
```

Comment:
This definitely should use `^` notation, and probably be `^3.7.7`

Reply:
Sure, I just wanted to stick to existing patterns :)